resolves #3994 fix instructions for Confluence XHTML migration (#3995)
asciidoctor/asciidoctor: docs/modules/migrate/pages/confluence-xhtml.adoc

. Save the script contents to a `convert.groovy` file in a working directory.
. Make the file executable according to your specific OS requirements.

> . Create a `html` directory for input files and a `asciidoc` directory for output files, both inside the working directory.

Reviewer: ```suggestion
. Create an `html` directory for input files and an `asciidoc` directory for output files, both inside the working directory.
```

Author: I was doubtful about "a 'html'" vs. "an 'html'" Thanks for clarifying!

> . Place individual files, or a directory containing files into the aforementioned `html` directory.

Reviewer: ```suggestion
. Place individual files, or a directory containing files, into the aforementioned `html` directory.
```

. Run `groovy convert` to convert the files contained inside the `html` directory.

> . Confirm the output file located in the `asciidoc` directory meets requirements.

Reviewer: ```suggestion
. Look for the generated output file in the `asciidoc` directory and confirm it meets your requirements.
```
